web-services/deploy/application/src/main/wildfly/add-datawave-configuration.cli

```diff
@@ -88,6 +88,7 @@ module add --name=com.mysql.driver --dependencies=javax.api,javax.transaction.ap
 /system-property=dw.metrics.reporter.port:add(value=${metrics.reporter.port})
 /system-property=dw.metrics.reporter.class:add(value=${metrics.reporter.class})
 /system-property=dw.cache.tableNames:add(value="${tables.to.cache}")
+/system-property=dw.model.defaultTableName:add(value=${table.name.metadata})
```

Comment:
I don't see a change to remove the property in `remove-datawave-configuration.cli`. Is it here and I missed it?

Reply:
Let me look.

Reply:
I tried to keep the alphabetizing scheme in tact: 

/system-property=dw.metrics.pool.admin.size:remove
/system-property=dw.model.defaultTableName:remove

Reply:
OK.  I see what happened.  The modification to the remove-datawave-configuration.cli was erased when I built the project again.
I will get that back in.  I did not know that could happen.